src/Tool/Newsletter.php

```diff
@@ -204,7 +204,12 @@ public static function sendNewsletterDocumentBasedMail(Mail $mail, SendingParamC
 
     protected static function obfuscateEmail(string $email): string
     {
-        return substr_replace($email, '.xxx', strrpos($email, '.'));
+        $offset = strrpos($email, '.');
+        if ($offset !== false) {
+            return substr_replace($email, '.xxx', $offset);
```

Comment:
should it be like this?
```suggestion
            return substr_replace($email, '.xxxx', $offset);
```

actually [.xxx](https://en.wikipedia.org/wiki/.xxx) is a valid top level domain 😅 
so no partners/projects working in the adult movie industry yet so far